tools/scripts/mysql_migration.py

```
@@ -1227,12 +1237,776 @@ def execute(self) -> tuple[int, list]:
+                values.append(f"('{record_id}', '{model_name_escaped}', '{provider_id}', "
+                            f"'{instance_id}', '{mt_escaped}', '{status_escaped}', "
+                            f"'{extra_escaped}', "
+                            f"{current_ts * 1000}, FROM_UNIXTIME({current_ts}), "
+                            f"{current_ts * 1000}, FROM_UNIXTIME({current_ts}))")
+
+            insert_sql = f"""
+                INSERT INTO tenant_model
+                (id, model_name, provider_id, instance_id, model_type, status, extra,
+                 create_time, create_date, update_time, update_date)
+                VALUES {', '.join(values)}
+            """
```

Ruff (0.15.20)

[error] Possible SQL injection vector through string-based query construction

(S608)